src/styles/review.css

```diff
@@ -0,0 +1,29 @@
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+}
```

Comment:
same as the buy.css file. you have the separate file but your naming suggests that review is an element of card block. `card__review` => review is an element of card.